### PR TITLE
Update sha in remotix-agent

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by suschizu** [comment]
just the bundle version changed this time

**Comment by suschizu** [comment]
`Error: Some applications are still running, add them to uninstall quit:
com.nulana.rxagentmac`
already added

File: Casks/remotix-agent.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'remotix-agent' do
   version '1.3.2'
```
Comment (glensc) [inline Casks/remotix-agent.rb]: version on webpage says 6.1:
- https://remotixcloud.com/downloads/

Comment (glensc) [reply Casks/remotix-agent.rb]: EDIT: pardon, for agent it still says 1.3.2

**Pull request merged**